src/xcode/ENA/ENAUITests/HealthCertificate/ENAUITests_20_CertificateReissuance.swift

```diff
@@ -35,17 +35,22 @@ class ENAUITests_CertificateReissuance: CWATestCase {
 
 		// Navigate to the person screen
 		app.cells[AccessibilityIdentifiers.HealthCertificate.Overview.healthCertifiedPersonCell].waitAndTap()
-		
-		app.swipeUp(velocity: .slow)
+
+		snapshot("screenshot_health_certificate_reissuance_refresh_certificate")
 
 		// Certificate Screen
 		app.cells[AccessibilityIdentifiers.HealthCertificate.Reissuance.cell].waitAndTap()
+		snapshot("screenshot_health_certificate_reissuance_consentScreen_1")
+		app.swipeUp(velocity: .slow)
+		snapshot("screenshot_health_certificate_reissuance_consentScreen_2")
```

Comment:
One Screenshot of the consent should be enough. In the past we removed all this scroll and screenshot. 

- This takes quite a while for the app to "settle" which will increase the CI time
- Only the first screenshot got used in the past

Reply:
This i did not know, i will fix it.